variables.tf

```diff
@@ -111,4 +177,45 @@ locals {
     for mysql_flexible_server_firewall_rule in keys(var.mysql_flexible_server_firewall_rule) :
     mysql_flexible_server_firewall_rule => merge(local.default.mysql_flexible_server_firewall_rule, var.mysql_flexible_server_firewall_rule[mysql_flexible_server_firewall_rule])
   }
+
+  /**
+    compare and merge custom and default values
+  */
+  postgresql_flexible_server_values = {
+    for postgresql_flexible_server in keys(var.postgresql_flexible_server) :
+    postgresql_flexible_server => merge(local.default.postgresql_flexible_server, var.postgresql_flexible_server[postgresql_flexible_server])
+  }
+
+  /**
+    deep merge of all custom and default values
+  */
+  postgresql_flexible_server = {
```

Comment:
don't duplicate the compare block again so please move this to line 180

Reply:
as discussed, is obsolete